<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>School Manager System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Bắt đầu thêm CSS tùy chỉnh tại đây nếu muốn */
QWidget {
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 14px;
}

QGroupBox {
    font-size: 16px;
    font-weight: bold;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    margin-top: 10px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px 5px 5px;
    left: 10px;
    color: #005A9E;
}

QPushButton {
    background-color: #007BFF;
    color: white;
    font-weight: bold;
    border: none;
    border-radius: 5px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #0056b3;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="header_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="student_info_box">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>163</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Thông tin học sinh</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_name">
            <property name="text">
             <string>&lt;b&gt;Họ tên:&lt;/b&gt; Nguyễn Văn An</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_gender">
            <property name="text">
             <string>&lt;b&gt;Giới tính:&lt;/b&gt; Nam</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_class">
            <property name="text">
             <string>&lt;b&gt;Lớp:&lt;/b&gt; 10A1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="parent_account_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_welcome">
            <property name="text">
             <string>Xin chào, &lt;b&gt;phuhuynh_an&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_change_password">
            <property name="text">
             <string>Đổi mật khẩu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_logout">
            <property name="styleSheet">
             <string notr="true">background-color: #E0E0E0; color: #333333;</string>
            </property>
            <property name="text">
             <string>Đăng xuất</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grades_box">
      <property name="title">
       <string>Bảng điểm tổng hợp</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_hk1">
          <attribute name="title">
           <string>Học kỳ 1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="table_hk1">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <column>
              <property name="text">
               <string>Môn học</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Miệng</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>15 phút</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>1 tiết</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Giữa kỳ</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Cuối kỳ</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Trung bình</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_hk2">
          <attribute name="title">
           <string>Học kỳ 2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableWidget" name="table_hk2">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <column>
              <property name="text">
               <string>Môn học</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Miệng</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>15 phút</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>1 tiết</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Giữa kỳ</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Cuối kỳ</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Trung bình</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_year">
          <attribute name="title">
           <string>Cả năm</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>6</x>
             <y>0</y>
             <width>1211</width>
             <height>231</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableWidget" name="table_year">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <column>
               <property name="text">
                <string>Môn học</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>TB HK1</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>TB HK2</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>TB Cả năm</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="xeploai">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Xếp loại: &lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Giỏi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGraphicsView" name="chartViewPlaceholder">
                <property name="toolTip">
                 <string>Khu vực này sẽ hiển thị biểu đồ so sánh điểm</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="comments_box">
      <property name="title">
       <string>Nhận xét của giáo viên</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="table_comments">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Môn học</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nhận xét</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="footer_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_copyright">
         <property name="styleSheet">
          <string notr="true">color: #888888; font-style: italic;</string>
         </property>
         <property name="text">
          <string>School Manager System</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="placeholder_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
